tests/ragger/conftest.py

```diff
@@ -36,6 +39,13 @@ def firmware(arg_model: str) -> Firmware:
             return fw
     raise ValueError("Unknown device model \"%s\"" % (arg_model))
 
+@pytest.fixture
+def cal_key() -> Optional[SigningKey]:
```

Comment:
Mmmh not really a fixture as it's not really a "parameter" of the test
I would suggest making a class out of it, also I suggest taking a look into the SigningAuthority defined in the exchange app tests, I instantiate and manipulate the same kind of structure. Maybe unify both and put it in ragger utils ?

Reply:
Indeed, should be better now :+1: